.github/workflows/release-layer-python.yml

```diff
@@ -0,0 +1,79 @@
+name: "Release Python Lambda Layer"
+
+on:
+  # (Using tag push instead of release to allow filtering by tag prefix.)
+  push:
+    tags:
+      - layer-python/**
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  build-layer:
+    runs-on: ubuntu-latest
+    outputs:
+      PYTHON_SDK_VERSION: ${{ steps.save-python-sdk-version.outputs.PYTHON_SDK_VERSION}}
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v3
+        with:
+          python-version: '3.9'
+
+      - name: Configure aws
+        run: aws configure set region us-west-1
+
```

Comment:
```suggestion
```

I don't think this is needed anymore.

Reply:
Not sure about that, I removed that step and the workflow [failed](https://github.com/ocelotl/opentelemetry-lambda/actions/runs/4964002577/jobs/8883739381#step:4:18).

Reply:
Why does it still need AWS access to build? I thought it was just doing a zip now.

Reply:
Ok, I refactored the script to only require `region` if it is publishing the layer by itself.